Title: Emily Wheeler
Date: 2015-06-01
Position: Graduate Student
Position_suffix: (Ph.D.)
Affiliation: Biomedical Sciences
Email: ecwheele at ucsd.edu
Fellowship: NSF GRFP
Alumni_or_current: Alumni
Headshot: /images/people/emily_wheeler.jpg
Template: person
Current_position: Postdoc in Zuzana Tothova lab at Harvard Medical School
End_date: 2020-08-04
<!-- Status: draft -->

## Degrees

B.S. Environmental Science, Indiana University 2014<br>

## Summary

Emily is a PhD student in the Biomedical Science Department at UCSD and a member of the genetics training program. Her thesis work in Gene Yeo’s lab focuses on using computational and molecular biology tools to uncover how protein-RNA interactions in cells play a role in cancer and neurodegeneration. Specifically, she is working on uncovering novel RNA binding proteins that are essential to form protein-aggregates in motor neurons that cause cell death in ALS. Additionally, she is developing computational methods to determine how point mutations in splicing factors that occur in blood cancer alter their interaction with cellular RNA. The results of this work will lead to a greater understanding of how these deadly diseases form, and therefore help design therapeutics to reverse the disease. 

Emily received her undergraduate degree from Indiana University where she completed a senior honors thesis on RNA editing in C. elegans in the lab of Dr. Heather Hundley. Emily is an NSF GRFP fellow and has received funding from the UCSD Genetics Training Program. She has also been an ARCS scholar since 2017. Emily developed and implemented a new course in computational methods that has been included in the core curriculum for the BMS graduate program at UCSD. She also served as the lead bioinformatics instructor for the Cold Spring Harbor course in Single Cell Analyses in 2018. Emily is also a tutor for the Reality Changers program in San Diego working with high school students who will become the first members of their family to attend college. Outside of lab, Emily spends her time in the mountains skiing during the winter months and climbing in the summer. 


## Publications

* **Wheeler EC** §, Vu A §, Einstein JM, DiSalvo M, Ahmed N, Van Nostrand EL, Shiskin AA, Jin W, Allbrighton NL, Yeo GW. Pooled CRISPR screens with image-based phenotyping on microRaft arrays reveals stress granule-regulatory factors. Nature Methods, 2020. (§ co-first author)

* Van Nostrand EL, Pratt GA, Yee BA, **Wheeler E**, Blue SM, Mueller J, Park SS, Garcia KE, Gelboin-Burkhart C, Nguyen TB, Rabano I, Stanton R, Sundararaman B, Wang R, Fu X-D, Graveley BR, Yeo GW. Principles of RNA processing from analysis of enhanced CLIP maps for 150 RNA binding proteins. Genome Biology, 2020.

*  Tan FE §, Sathe S §, **Wheeler EC**, Nussbacher JK, Peter S, Yeo GW . A Transcriptome-wide Translational Program Defined by LIN28B Expression Level.  Mol Cell. 2018 Dec (§ co-first author )

* Transcriptome–pathology correlation identifies interplay between TDP‑43 and the expression of its kinase CK1E in sporadic ALS. Krach F, Batra R, **Wheeler EC**, Vu AQ, Wang R, Hutt K, Rabin SJ, Baughn MW, Libby RT, Diaz‑Garcia S, Stauffer J, Pirie E, Saberi S, Rodriguez M, Madrigal AA, Kohl Z, Winner B, Yeo GW, Ravits J. Acta Neuropathologica, 2018. 

* The C. elegans neural editome reveals an ADAR target mRNA required for proper chemotaxis. Deffit SN, Yee BA, Manning AC, Rajendren S, Vadlamani P, **Wheeler EC**, Domissy A, Washburn MC, Yeo GW, Hundley HA. Elife. 2017 Sep 19.

* **Wheeler EC**, Van Nostrand EL, Yeo GW
Advances and challenges in the detection of transcriptome-wide protein–RNA interactions WILEY INTERDISCIPLINARY REVIEWS: RNA. 2017

*  Carter H, Marty R, Hofree M, Gross AM, Jensen J, Fisch KM, Wu X, DeBoever C, Van Nostrand EL, Song Y, **Wheeler E**, Kreisberg JF, Lippman SM, Yeo GW, Gutkind JS, and Ideker T. Interaction landscape of inherited polymorphisms with somatic events in cancer. Cancer Discovery. 2017.
 
* Batra R, Stark TJ, Clark E, Belzile JP, **Wheeler EC**, Yee BA, Huang H, Gelboin-Burkhart C, Huelga SC, Aigner S, Roberts BT, Bos TJ, Sathe S, Donohue JP, Rigo F, Ares M Jr, Spector DH, Yeo GW. RNA-binding protein CPEB1 remodels host and viral RNA landscapes. Nat Struct Mol Biol. 2016 Dec;23(12):1101-1110. 
 
 
*  **Wheeler E**, Washburn M, Major F, Rusch D, Hundley H. “Noncoding regions of C. elegans mRNA undergo selective adenosine to inosine deamination and contain a small number of editing sites per transcript. RNA Biology 2015;12(2):162-74.

* Washburn M, Kakaradov B, Sundararaman B, **Wheeler E**, Hoon S, Yeo G, Hundley H. “The dsRBP and inactive editor ADR-1 utilizes dsRNA binding to regulate A-to-I RNA editing across the C. elegans transcriptome.” Cell Reports 2014 Feb 27;6(4):599-607.

* Hin H, **Wheeler E**, Phillips R. “Root-induced changes in nutrient cycline in forests depends on exudation rates.” Siol Biology and Biochemistry 2014 Nov;78:213-221.
