Title: Eric Van Nostrand
Date: 2012-10-05
Position: Post-Doctoral Fellow
Affiliation: Cellular and Molecular Medicine
Email: elvannostrand@ucsd.edu
Fellowship:
Headshot: /images/people/eric_van_nostrand.jpg
Template: person
Alumni_or_current: Current

## Degrees
Ph.D. (Genetics), Stanford University School of Medicine, 2012<br>
S.B. (Biology) Massachusetts Institute of Technology, 2005<br>




## Summary

Eric Van Nostrand received his S.B. in Biology from MIT in 2005, with research experience in Dr. Chris Burge’s lab on computational and experimental analysis of alternative splicing regulation. After spending a year with Dr. Fred Gage & Dr. Gene Yeo at the Salk Institute for Biological Studies, Eric attended Stanford University for graduate school, joining Dr. Stuart Kim’s lab. Eric’s thesis work focused on genomic, genetic, and molecular analysis of aging in *C. elegans*, identifying novel regulators of transcriptional changes with age that revealed links between developmental regulators and aging. As a part of this work, Eric identified transcription factor complexity (defined as the number of factors that bind to the same genomic loci) as a significant feature for biological interpretation of ChIP-seq data. Low-complexity binding sites tended to correlate with factor-responsive target expression, whereas analysis of HOT (highly occupied target regions) bound by 65% or more of factors revealed a potential novel regulatory mechanism for highly expressed, essential, ubiquitous genes. In 2013, Eric joined the Yeo lab to perform postdoctoral research on general RNA binding protein regulatory networks as well as identification and characterization of critical RNA binding proteins in neuronal stem cell maintenance and differentiation.

## Publications
* **Van Nostrand EL**, Shishkin AA, Pratt GA, Nguyen TB, Yeo GW. Variation in single-nucleotide sensitivity of eCLIP derived from reverse transcription conditions. Methods, 2017. [[PDF](/papers/2017/Eric_singel_nucelotide_Methods_2017.pdf)]

* **Van Nostrand EL**, Nguyen TB, Gelboin-Burkhart C, Wang R, Blue SM, Pratt GA, Louie AL, Yeo GW. Robust, Cost-Effective Profiling of RNA Binding Protein Targets with Single-end Enhanced Crosslinking and Immunoprecipitation (seCLIP). Methods Mol Biol., 2017. [[PDF](/papers/2017/Eric_Methods_MolBiol_2017.pdf)]   

* **Van Nostrand EL**, Gelboin-Burkhart C, Wang R, Pratt GA, Blue SM, Yeo GW. 
CRISPR/Cas9-mediated integration enables TAG-eCLIP of endogenously tagged RNA binding proteins. Special issue “Protein-RNA: Structure function and recognition” in Methods, 2016.[[PDF](/papers/2016/VanNostrand_Methods2016.pdf)]

* **Van Nostrand EL**, Pratt GA, Shishkin AA, Gelboin-Burkhart C, Fang M, Sundararaman B, Blue SM, Nguyen TB, Surka C, Elkins K, Stanton R, Rigo F, Guttman M, Yeo GW.   
Enhanced CLIP (eCLIP) enables robust and scalable transcriptome-wide discovery and characterization of RNA binding protein binding sites. 
Nature Methods (2016)[[PDF](/papers/2016/nmeth_eric_2016.pdf)]   


* Conway AE, **Van Nostrand EL**, Pratt GA, Aigner S, Wilbert ML, Sundararaman B, Freese P, Lambert NJ, Sathe S, Liang TY, Essex A, Landais S, Burge CB, Jones DL, Yeo GW.   
Enhanced CLIP uncovers IMP protein-RNA targets in human pluripotent stem cells important for cell adhesion and survival. 
 Cell Reports (2016) [[PDF](/papers/2016/nmeth_eric_2016.pdf)] 

* **Van Nostrand, E.L.** & Kim, S.K. Integrative analysis of C. elegans modENCODE ChIP-seq datasets to infer transcription factor-responsive targets and upstream regulators of differentially-expressed genes from expression profiling experiments. Genome Research (26, 2013, doi:10.1101/gr.152876.112). [[PMID](http://genome.cshlp.org/content/early/2013/03/25/gr.152876.112.abstract)]

* **Van Nostrand, E.L.**, Sanchez-Blanco, A., Wu, B., Nguyen, A., & Kim, S.K. Roles of the Developmental Regulator unc-62/Homothorax in Limiting Longevity in Caenorhabditis elegans. PLoS Genet. (2013); 9(2): e1003325 [[PMID](http://www.ncbi.nlm.nih.gov/pubmed/23468654)]

* **Van Nostrand, E.L.** & Kim, S.K. Seeing elegance in gene regulatory networks of the worm.  Curr Opin Genet Dev. (2011) Dec;21(6):776-86.  [[PMID](http://www.ncbi.nlm.nih.gov/pubmed/21963133)]

* Gerstein, M.B.* , Lu, Z.J.* , **Van Nostrand, E.L*** , Cheng, C. * , Arshinoff, B.I. * , Liu, T.* , Yip, K.Y.* , Robilotto, R.* , Rechtsteiner, A.* , Ikegami, K.* , Alves, P. * , Chateigner, A.* , Perry, M.* , Morris, M.* , Auerbach, R.K.* , Feng, X.* , Leng, J.* , Vielle, A.*  , Niu, W.* , Rhrissorrakrai, K.* , et al. Integrative Analysis of the Caenorhabditis elegans Genome by the modENCODE Project. Science (2010); 330(6012). 1775-87 (*Joint first authors) [[PMID](http://www.ncbi.nlm.nih.gov/pubmed/21177976)]

* **Van Nostrand, E.L.** & Yeo, G.W. Evolutionarily Conserved Intronic Splicing Regulatory Elements in the Human Genome. Encyclopedia of Life Sciences. (2008) [[PMID](http://onlinelibrary.wiley.com/doi/10.1002/9780470015902.a0021005/full)]

* Yeo, G.W., **Van Nostrand E.L.**, Liang T.Y. Discovery and analysis of evolutionarily conserved intronic splicing regulatory elements. PLoS Genet. (2007); 3(5) [[PMID](http://www.ncbi.nlm.nih.gov/pubmed/17530930)]

* Wang, Z., Xiao, X., **Van Nostrand, E.**, & Burge, C.B. General and specific functions of exonic splicing silencers in splicing control. Mol Cell. (2006) 23. 61-70. [[PMID](http://www.ncbi.nlm.nih.gov/pubmed/16797197)]

* Yeo, G.W., **Van Nostrand, E.**, Holste, D., Poggio, T., & Burge, C.B. Identification and analysis of alternative splicing events conserved in human and mouse. PNAS. (2005) 102. 2850-2855 [[PMID](http://www.ncbi.nlm.nih.gov/pubmed/15708978)]
	