Title: Shashank Sathe
Date: 2014-06-01
Position: Staff
Position_suffix: Bioinformatics Analyst
Affiliation:
Email: shsathe at ucsd.edu
Fellowship:
Headshot: /images/people/shashank_sathe-2.jpg
Template: person
Alumni_or_current: Current
<!-- Status: draft -->

## Degrees
B.Tech, Bioinformatics, D.Y.Patil University, Mumbai, India, 2010<br>
M.S, Bioinformatics, San Diego State University, San Diego, USA, 2015<br>


## Summary

Shashank received his Bachelor of Technology degree in Bioinformatics from the Dr.D.Y.Patil University in Mumbai, India in 2010. He moved to USA to pursue graduate studies in Bioinformatics at the San Diego State University in 2011 and achieved his Master's degree in 2015.

Shashank started his Bioinformatics career in 2013 as an Intern at the Allen Lab in J.Craig Venter Institute, where he performed transcriptomic analyses on marine species such as diatoms. In 2014, Shashank joined Dr. Yeo's group as a Bioinformatics staff member, focused on analyzing RNA processing data using computational methods.

Currently, Shashank works on studying RNA subcellular localization and RNA-RBP interactions under stress, developmental and diseased conditions.

## Publications

* Luo EC, Nathanson JL, Tan FE, Schwartz JL, Schmok JC, Shankar A, Markimiller S, Yee BA, **Sathe S**, Pratt GA, Scaletta DB, Ha Y, Hill DE, Aigner S, Yeo GW. Large-scale tethered function assays identify factors that regulate mRNA stability and translation.  Nature Structural and Molecular Biology. 2020.

* Lorenz DA  §, **Sathe S**  §, Einstein JM, Yeo GW. Direct RNA sequencing enables m6A detection in endogenous transcript isoforms at base specific resolution. RNA. 2019 (§ co-first author )

* Tan FE §, **Sathe S** §, Wheeler EC, Nussbacher JK, Peter S, Yeo GW . A Transcriptome-wide Translational Program Defined by LIN28B Expression Level.  Mol Cell. 2018 Dec (§ co-first author )

* Lardelli RM, Schaffer AE, Eggens VR, Zaki MS, Grainger S, **Sathe S**, Van Nostrand EL, Schlachetzki Z, Rosti B, Akizu N, Scott E, Silhavy JL, Heckman LD,
Rosti RO, Dikoglu E, Gregor A, Guemez-Gamboa A, Musaev D, Mande R, Widjaja A, Shaw TL, Markmiller S, Marin-Valencia I, Davies JH, de Meirleir L, Kayserili H,
Altunoglu U, Freckmann ML, Warwick L, Chitayat D, Blaser S, Çağlayan AO, Bilguvar K, Per H, Fagerberg C, Christesen HT, Kibaek M, Aldinger KA, Manchester D,
Matsumoto N, Muramatsu K, Saitsu H, Shiina M, Ogata K, Foulds N, Dobyns WB, Chi NC, Traver D, Spaccini L, Bova SM, Gabriel SB, Gunel M, Valente EM, Nassogne MC, 
Bennett EJ, Yeo GW, Baas F, Lykke-Andersen J, Gleeson JG. Biallelic mutations in the 3' exonuclease TOE1 cause pontocerebellar hypoplasia and uncover a role in
snRNA processing. Nat Genet. 2017 Mar;49(3):457-464. doi: 10.1038/ng.3762.

* Batra R, Stark TJ, Clark E, Belzile JP, Wheeler EC, Yee BA, Huang H, Gelboin-Burkhart C, Huelga SC, Aigner S, Roberts BT, Bos TJ, **Sathe S**, Donohue JP, Rigo F, Ares M Jr, Spector DH, Yeo GW. RNA-binding protein CPEB1 remodels host and viral RNA landscapes. Nat Struct Mol Biol. 2016 Dec;23(12):1101-1110. doi:
10.1038/nsmb.3310. 

* Conway AE, Van Nostrand EL, Pratt GA, Aigner S, Wilbert ML, Sundararaman B, Freese P, Lambert NJ, **Sathe S**, Liang TY, Essex A, Landais S, Burge CB, Jones DL, Yeo GW. Enhanced CLIP Uncovers IMP Protein-RNA Targets in Human Pluripotent Stem Cells Important for Cell Adhesion and Survival. Cell Rep. 2016 Apr 19;15(3):666-679.


* Gerson-Gurwitz A, Wang S, **Sathe S**, Green R, Yeo GW, Oegema K, Desai A. A Small RNA-Catalytic Argonaute Pathway Tunes Germline Transcript Levels to Ensure
Embryonic Divisions. Cell. 2016 Apr 7;165(2):396-409. doi: 10.1016/j.cell.2016.02.040.


* Lipson DA, Haggerty JM, Srinivas A, Raab TK, **Sathe S**, Dinsdale EA. Metagenomic insights into anaerobic metabolism along an Arctic peat soil profile. PLoS One.
2013 May 31;8(5):e64659. doi: 10.1371/journal.pone.0064659. 
